src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/signin/AttachedSignInPage.java

```diff
@@ -62,6 +64,14 @@ public SignInPage typeUsername(String username) {
     return new FormError().getError();
   }
 
+  @Override public boolean isPasswordMasked() {
+    return passwordField.getAttribute("type").equals("password");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "password" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)